mathics/core/parser/tokeniser.py

```diff
@@ -206,6 +206,9 @@
     ('Therefore', r' \u2234 '),
     ('Because', r' \u2235 '),
     ('Backslash', r' \u2216 '),
+
+    ('Information', r'\?\?'),
+    ('Definition', r'\?'),
```

Comment:
conflicts with `PatternTest` token.

Reply:
In[1]:= a ? b//FullForm
Out[1]= PatternTest[a, b]

In[2]:= ? a b //FullForm
Out[2]= Times[b, Information[a, Rule[LongForm, False]]]


In Wolfram's

 In[1]:= a ? b //FullForm
 Out[1]//FullForm= FullForm[a?b]

 In[2]:= ? a b //FullForm
Global`a
Global`b 
 Out[2]//FullForm= FullForm[Null]